lib/lnc.ts

```diff
@@ -195,7 +195,11 @@ export default class LNC {
         );
 
         // add an event listener to disconnect if the page is unloaded
-        window.addEventListener('unload', this.wasm.wasmClientDisconnect);
+        if (typeof window !== "undefined") {
+            window.addEventListener('unload', this.wasm.wasmClientDisconnect);
```

Comment:
here we can add an eventListener only when the `window` is available. other contexts might need to manually register some custom code to disconnect.

Reply:
Good call on this one